.github/workflows/Benchmarks.yml

```
@@ -0,0 +1,19 @@
+name: Benchmarks
+on:
+  pull_request_target:
+    branches: [ master ]
+permissions:
+  pull-requests: write
+
+jobs:
+  bench:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version:
+          - 'lts'
+          - '1'
+    steps:
+      - uses: MilesCranmer/AirspeedVelocity.jl@action-v1
+        with:
+          julia-version: ${{ matrix.version }}
```

```suggestion
          julia-version: ${{ matrix.version }}
          bench-on: ${{ github.event.pull_request.head.sha }}
```
I recommend using the `bench-on` parameter. For more details, see https://github.com/MilesCranmer/AirspeedVelocity.jl/issues/97#issuecomment-2935866653.